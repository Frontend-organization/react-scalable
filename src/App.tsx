import { hot } from 'react-hot-loader/root'
import Home from './pages/Home'

const App: React.FC = () => {
  return <Home />
}

export default hot(App)
