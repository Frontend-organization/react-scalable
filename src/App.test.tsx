import { render } from '@testing-library/react'

import App from './App'

describe('<App />', () => {
  it('should render correctly by snapshot', () => {
    const { getByAltText } = render(<App />)
    expect(<App />).toMatchSnapshot()
    expect(getByAltText('initial-build')).toBeInTheDocument()
  })
})
