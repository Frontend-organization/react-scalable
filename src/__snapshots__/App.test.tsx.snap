// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render correctly by snapshot 1`] = `<App />`;
